tutorials/polkadot-sdk/build-a-blockchain/simulate-a-network.md

```diff
@@ -0,0 +1,197 @@
+    ```bash
+    ./target/release/solochain-template-node \
+    --base-path /tmp/bob \
+    --chain local \
+    --bob \
+    --port 30334 \
+    --rpc-port 9946 \
+    --node-key 0000000000000000000000000000000000000000000000000000000000000002 \
+    --validator \
+    --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp
+    ```
+
+    Key differences in this command:
+    
+    1. Unique paths and ports - to avoid conflicts on the same machine, different values are used for:
+        - `--base-path` - set to `/tmp/bob`
+        - `--port` - set to `30334`
+        - `--rpc-port` - set to `9946`
+
+    2. Bootnode specification - the `--bootnodes` option is crucial for network discovery:
+        - Format - `/ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp`
+        - Components:
+            - `ip4` - indicates IPv4 format
+            - `127.0.0.1` - IP address of the running node (localhost in this case)
+            - `tcp` - specifies TCP for peer-to-peer communication
+            - `30333` - port number for peer-to-peer TCP traffic
+            - `12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp` - unique identifier of the `Alice` node
+
+## Verify Blocks are Produced and Finalized
+
+After starting the second node, both nodes should connect as peers and commence block production.
+
+Follow these steps to verify that blocks are being produced and finalized:
+
+1. Observe the output in the terminal of the first node (`Alice`):
+
+    --8<-- 'code/tutorials/polkadot-sdk/build-a-blockchain/simulate-a-network/node-output-1.html'
+
+    Key information in this output:
+
+    - Second node discovery - `discovered: 12D3KooWHdiAxVd8uMQR1hGWXccidmfCwLqcMpGwR6QcTP6QRMuD`
+    - Peer count - `1 peers`
+    - Block production - `best: #4 (0xe176…0430)`
+    - Block finalization - `finalized #1 (0x75bb…e82d)`
+
+2. Check the terminal of the second node (`Bob`) for similar output
+
+3. Shut down one node using `Control-C` in its terminal. Observe the remaining node's output:
+
+    <div id='termynal' data-termynal>
+        <span data-ty>2024-09-10 09:10:03 💤 Idle (1 peers), best: #51 (0x0dd6…e763), finalized #49 (0xb70a…1fc0), ⬇ 0.7kiB/s ⬆ 0.6kiB/s</span>
+        <span data-ty>2024-09-10 09:10:08 💤 Idle (0 peers), best: #52 (0x2c40…a50e), finalized #49 (0xb70a…1fc0), ⬇ 0.3kiB/s ⬆ 0.3kiB/s</span>
+    </div>
+
+    Note that the peer count drops to zero, and block production stops.
+
+4. Shut down the second node using `Control-C` in its terminal
+
+5. Clean up chain state. To remove the chain state from the simulated network, use the `purge-chain` subcommand:
+
+    - For Alice's node:
+        ```bash
+        ./target/release/solochain-template-node purge-chain --base-path /tmp/alice --chain local -y
+        ```
+    - For Bob's node:
+        ```bash
+        ./target/release/solochain-template-node purge-chain --base-path /tmp/bob --chain local -y
+        ```
```

Comment:
Can you please break these up into several lines so you don't have to horizontally scroll to see the entire command